server/index.ts

```diff
@@ -54,7 +54,7 @@ server.start().then((): void => {
     });
 
     // serve homepage
-    app.use('/', (req, res) =>
+    app.all('/', (req, res) =>
```

Comment:
What's the difference here between `all` and `use`?

Reply:
This was because when it was `app.use()`, all requests to '/' were serving html. With `app.all()`, only the requests that reach the end of the middleware chain serve the html.